package.json

```diff
@@ -52,7 +52,8 @@
         "overrides": {
             "trim@<0.0.3": ">=0.0.3",
             "trim-newlines@<3.0.1": ">=3.0.1",
-            "glob-parent@<5.1.2": ">=5.1.2"
+            "glob-parent@<5.1.2": ">=5.1.2",
+            "minimatch@<3.0.5": "3.0.5"
```

Comment:
I thought the specific one is better because that is the only thing we can validate for now. 
Btw, I am not happy with the PR because it changed more in the lock file that I expected.